pkgs/tools/admin/simp_le/default.nix

```diff
@@ -17,10 +18,13 @@ python3Packages.buildPythonApplication rec {
   '';
 
   checkPhase = ''
+    runHook preCheck
     $out/bin/simp_le --test
+    runHook postCheck
   '';
 
-  propagatedBuildInputs = with python3Packages; [ acme setuptools-scm josepy idna ];
+
+  propagatedBuildInputs = with python3Packages; [ acme setuptools-scm josepy idna setuptools ];
```

Comment:
Please move setuptools into nativeBuildInputs, unless simp_le imports `pkg_resources`.

Reply:
This is still unaddressed. I doubt that `setuptools` is a runtime dependency.

Reply:
well your doubt may be missguided:
setuptools in nativeinputs:
```
~/nixpkgs result/bin/simp_le 
Traceback (most recent call last):
  File "/nix/store/3lqjn67l5ncfp0a96rgc0qfgi21sjvb2-simp_le-client-0.20.0/bin/.simp_le-wrapped", line 6, in <module>
    from simp_le import main
  File "/nix/store/3lqjn67l5ncfp0a96rgc0qfgi21sjvb2-simp_le-client-0.20.0/lib/python3.11/site-packages/simp_le.py", line 41, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

Reply:
Five kinds of weird, but okay!